includes/Modules/Analytics.php

```diff
@@ -422,13 +422,9 @@ protected function handle_provisioning_callback() {
 	 * @return array Map of datapoints to their definitions.
 	 */
 	protected function get_datapoint_definitions() {
-		$shareable = Feature_Flags::enabled( 'dashboardSharing' );
 		// If ga4Reporting is enabled, the dashboard view controls which
 		// Analytics module is shareable.
-		if ( $shareable && Feature_Flags::enabled( 'ga4Reporting' ) ) {
-			$settings  = $this->get_settings()->get();
-			$shareable = self::DASHBOARD_VIEW === $settings['dashboardView'];
-		}
+		$shareable = ! Feature_Flags::enabled( 'ga4Reporting' ) || self::DASHBOARD_VIEW === $this->get_settings()->get()['dashboardView'];
```

Comment:
Preferably, we could make this code more readable by having an `if` block similar to the `Analytics_4::get_datapoint_definitions` method.